acrobat/blocks/dc-converter-widget/dc-converter-widget.js

```diff
@@ -317,7 +314,14 @@ export default async function init(element) {
   const verbIncludeList = ['compress-pdf', 'fillsign', 'sendforsignature', 'add-comment',
     'delete-pages', 'reorder-pages', 'split-pdf', 'insert-pdf', 'extract-pages', 'crop-pages', 'number-pages'];
 
-  if (verbIncludeList.includes(VERB) || preRenderDropZone) {
+  const INLINE_SNIPPET = widget.querySelector(':scope > section#edge-snippet');
+  if (INLINE_SNIPPET) {
+    widgetContainer.dataset.rendered = 'true';
```

Comment:
Hi @peller, we probably need a unit test for lines 319-323?

Reply:
done